### PR TITLE
Power shell operations #313

### LINKED ISSUE #313: PowerShell operations
Add basic PowerShell operations

### PR DESCRIPTION
**Pull request by gituser173**
Fixes #313 
Signed-off-by: Claudiu Ban claudiu-laurian.ban@hp.com

### CODE REVIEW

**Comment by gituser173** [comment]
@mihaitusa please review

File: content/io/cloudslang/powershell/execute_powershell_script.sl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#
+####################################################
+# Executes PowerShell script on a given remote host.
+#
+# Inputs:
+#   - host - the hostname or IP address of the PowerShell host
+#   - user_name - The username used to connect to the the remote machine.
+#   - password - The password used to connect to the remote machine.
+#   - script - The PowerShell script that will run on the remote machine.
+#   - winrm_enable_https - If set to true, HTTPS is used to connect to the WinRM server. Otherwise HTTP is used.
```
Comment (mihaitusa) [inline content/io/cloudslang/powershell/execute_powershell_script.sl]: @gituser173  I think the description "The default value is false." should be replaced with: - Default: False

File: content/io/cloudslang/powershell/execute_powershell_script.sl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        default: get('user_name', None)
+    - password
+    - script
+    - connectionType:
+        default: "'WINRM_NATIVE'"
+        overridable: false
+    - winrm_enable_https:
+        required: false
+    - winrmEnableHTTPS:
+        default: get('winrm_enable_https', None)
```
Comment (mihaitusa) [inline content/io/cloudslang/powershell/execute_powershell_script.sl]: @gituser173  I think here the get function should be: get('winrm_enable_https', False) if the default value is False.

**Comment by mihaitusa** [comment]
@gituser173  Please handle over the small comments. Thanks!

**Comment by gituser173** [comment]
@mihaitusa please review

**Pull request merged**